packages/storage-postgres/package.json

```diff
@@ -59,10 +59,13 @@
     "radash": "^12.1.0",
     "redlock": "^5.0.0-beta.2",
     "rsmq": "^0.12.4",
-    "uuid": "10.0.0"
+    "uuid": "10.0.0",
+    "steveo": "*"
```

Comment:
We don't want to lock `steveo` down to a specific version? It will break if there is a breaking change released in `steveo` 🤔

Reply:
It should auto resolve to whatever the version the consuming service is using